### PR TITLE
feat: add enable and disable feed manager queries

### PR DESCRIPTION
**Pull request by ChrisAmora**
https://smartcontract-it.atlassian.net/browse/DPA-1067
Update UI with Enable/Disable JD mutations.

Screenshot 2024-10-25 at 15 09 30
Screenshot 2024-10-25 at 15 09 47
Screenshot 2024-10-25 at 15 09 58
Screenshot 2024-10-25 at 15 10 09

- [x] This PR has an accompanying changeset if needed.

### CODE REVIEW

File: src/screens/FeedsManager/FeedsManagerScreen.tsx [hunk 1]
```diff
@@ -1,24 +1,84 @@
...
   const { id } = useParams<RouteParams>()
+  const { handleMutationError } = useMutationErrorHandler()
+  const dispatch = useDispatch()
 
   const { data, loading, error } = useFeedsManagerWithProposalsQuery({
     variables: { id },
     fetchPolicy: 'cache-and-network',
   })
 
+  const [enableFeedsManager] = useMutation<
```
Comment (graham-chainlink) [inline src/screens/FeedsManager/FeedsManagerScreen.tsx:72]: Interesting how come we dont use this ` const [enableFeedsManager, { data, loading, error }] = useMutation(...);`?

Then we can also use the <Loading /> with the loading right?

File: src/screens/FeedsManager/FeedsManagerScreen.tsx [hunk 2]
```diff
@@ -1,24 +1,84 @@
...
+    disableFeedsManager(id: $id) {
+      ... on DisableFeedsManagerSuccess {
+        feedsManager {
+          id
+          name
+          uri
+          publicKey
+          isConnectionActive
+          createdAt
+          disabledAt
```
Comment (graham-chainlink) [inline src/screens/FeedsManager/FeedsManagerScreen.tsx:47]: do we need to fetch the full information from feedsManager? eg do we need uri publickey etc?

File: src/screens/FeedsManager/FeedsManagerCard.test.tsx [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
 
 function renderComponent(manager: FeedsManagerFields) {
   renderWithRouter(
     <>
       <Route path="/">
-        <FeedsManagerCard manager={manager} />
+        <FeedsManagerCard
+          manager={manager}
+          onDisable={() => {}}
+          onEnable={() => {}}
```
Comment (graham-chainlink) [inline src/screens/FeedsManager/FeedsManagerCard.test.tsx:20]: Like how we have the test `navigate to edit`, do u think we could test clicking disable and enable will call onDisable and onEnable?

File: src/screens/FeedsManager/FeedsManagerScreen.tsx (hunk 1 above)
Comment (ChrisAmora) [reply src/screens/FeedsManager/FeedsManagerScreen.tsx:72]: I followed the model from [EditFeedsManagerScreen](https://github.com/smartcontractkit/operator-ui/blob/main/src/screens/EditFeedsManager/EditFeedsManagerScreen.tsx)
But with the loading would be better I think

File: src/screens/FeedsManager/FeedsManagerScreen.tsx (hunk 2 above)
Comment (ChrisAmora) [reply src/screens/FeedsManager/FeedsManagerScreen.tsx:47]: Yeah, this is the most up to date manager. In the rare case that someone update in the API directly the user would receive the updated manager.
I don't think first-level resolvers place a significant burden on the server either.

File: src/screens/FeedsManager/FeedsManagerScreen.tsx (hunk 1 above)
Comment (graham-chainlink) [reply src/screens/FeedsManager/FeedsManagerScreen.tsx:72]: ah interesting, looks good then haha

File: src/screens/FeedsManager/FeedsManagerScreen.tsx (hunk 2 above)
Comment (graham-chainlink) [reply src/screens/FeedsManager/FeedsManagerScreen.tsx:47]: ah okay make sense!

**Comment by graham-chainlink** [comment]
Also looks like [there are 2 issues](https://sonarqube.main.prod.cldev.sh/project/issues?issueStatuses=OPEN%2CCONFIRMED&severities=CRITICAL&inNewCodePeriod=true&pullRequest=92&id=smartcontractkit_operator-ui) in sonarqube , u might want to acknowledge them if we dont plan to action it.

**Review by graham-chainlink — DISMISSED** [review]

**Comment by ChrisAmora** [comment]
> Also looks like [there are 2 issues](https://sonarqube.main.prod.cldev.sh/project/issues?issueStatuses=OPEN%2CCONFIRMED&severities=CRITICAL&inNewCodePeriod=true&pullRequest=92&id=smartcontractkit_operator-ui) in sonarqube , u might want to acknowledge them if we dont plan to action it.

Fixed the issues.

**Review by giogam — APPROVED** [review]

**Review by jkongie — APPROVED** [review]

**Pull request merged**